template/5-mvp.md

```diff
@@ -1,34 +1,71 @@
+* Event participation - This is the most obvious feature, offering our users the chance to discover personalised events based on their passions and curiosity. They can easily find them thanks to a powerful search tool and profile customisation.
+* Getting to know people from the same background - Coming to EPFL can be difficult, especially if you don't know anyone. Echo is keen to encourage people from the same class or section to get to know each other, so students can organise events restricted to a certain milieu. 
+* Associations - At present, the associations are by far the most active in creating events on campus. However, they all have the same problem: a lack of participants. Echo enables associations to promote their events to users and reach their target audience more easily.
 
 ## Success Criteria
 
-*How will you evaluate the success of the MVP?*
+### Event Creation
 
-*Metrics include user penetration, quality / satisfaction.*
+To assess the success of Echo, we will be looking at the number of events officially held. We estimate that an event brings together an average of 10 people. Our application will be a success if we manage to organise at least 1 event per week for every 100 students. So, for a first-year computer science class of 600 students, at least 6 events should be organised within this class using our application. 
 
-*If applicable, progress in discussions with ecosystem partners / investors / customers.*
+### User Adoption & Engagement
 
-## Features Outside the Scope
+* In its first month at the start of the academic year, our application plans to reach at least 30% of new EPFL students (first-year, exchange and repeat students ~ 1,000 users), and at least 10% of the rest of the student body (~ 1,000 users). 
+* Since our events are generally organised on a weekly basis, we're mainly expecting weekly rather than daily users. Our aim is to achieve a DAU of 5% (100 users) and a MAU of 80% (1,600) in the first month of its launch.
+* We expect the application to be used a lot at the beginning of the semesters and then to be put to one side slightly at the end of the semester/exam periods. However, we still expect to be able to achieve a retention rate of at least 50%.
+
+### Feedback & Satisfaction
 
-*The MVP must be viable and minimal.*
+* We hope to achieve a score of 4 stars out of 5 in the Play Store ratings. We will be regularly monitoring feedback to identify recurring comments about our application, and adapting our approach accordingly.
+* A survey will be carried out at the end of each month to respond to requests from users who would like to add new categories, and to ensure that each of them has at least one event available per week. This will enable us to assess demand and, why not, try to improve the visibility of certain activities that are struggling to develop (special events organised by the application).
```

Comment:
maybe also a survey to carry out in order to know the user experience with the app so far and if they would recommend it to their peers or if they like the application enough to continue using it. 
Maybe include like time to time popups like we have on certain applications which ask the user for feedback by rating the app in order to know user feeling about the same.

Reply:
I'll take it like this ;) thanks for pointing this out !